documentation/book/proc-manual-restart-kafka.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka.adoc
+
+[id='proc-manual-restart-kafka-{context}']
+= Restarting a Kafka cluster manually
+
+This procedure describes how to manually restart an existing Kafka cluster by using an {ProductPlatformName} annotation.
```

Comment:
@ppatierno Do we want to say ```manually restart``` or ```manually rolling update```?

Reply:
`rolling restart`? That is what it really is, right ;-)

Reply:
Suggest we change the title to: _Performing a rolling update of a Kafka cluster_

Reply:
_Change the first sentence to align with the new terminology:_

This procedure describes how to manually restart a Kafka cluster by using an {ProductPlatformName} annotation. This triggers a rolling update of the pods inside the cluster.